### PR TITLE
Add discovering services in example

### PR DESCRIPTION
**Pull request by emmanuelgeoffray**
In the example, it showed how to publish a service but not how to lookup for one.
Fixed now with these commits.

### CODE REVIEW

File: example/config.make [hunk 1]
```diff
@@ -88,7 +88,7 @@
 # add a runtime path to search for those shared libraries, since they aren't 
 # incorporated directly into the final executable application binary.
 # TODO: should this be a default setting?
-# PROJECT_LDFLAGS=-Wl,-rpath=./libs
+PROJECT_LDFLAGS=-lz -L /lib64 -l avahi-client -lavahi-common
```
Comment (arturoc) [inline example/config.make]: it's better it fhis flags went in an addon_config.mk file. if you want to take a look that would be great. if you don't know very well the structure i can take a look at some point.

thanks!!

Comment (emmanuelgeoffray) [inline example/config.make]: yes! my mate @hugohil fixed this.
best,
e

**Comment by arturoc** [comment]
great, can you remove it from the config.make file then?

**Comment by emmanuelgeoffray** [comment]
sure.
done!

**Comment by arturoc** [comment]
thanks!

**Pull request merged**